beta/src/pages/learn/conditional-rendering.md

```diff
@@ -701,32 +702,32 @@ function Drink({ name }) {
 export default function DrinkList() {
   return (
     <div>
-      <Drink name="tea" />
-      <Drink name="coffee" />
+      <Drink name="herbata" />
+      <Drink name="kawa" />
     </div>
   );
 }
 ```
 
 </Sandpack>
 
-Here the information about each drink is grouped together instead of being spread across multiple conditions. This makes it easier to add more drinks in the future.
+Tutaj informacje o każdym napoju są pogrupowane razem, a nie rozłożone na kilka warunków. Ułatwi to dodawanie kolejnych napojów w przyszłości.
 
-Another solution would be to remove the condition altogether by moving the information into objects:
+Innym rozwiązaniem byłoby całkowite usunięcie warunku i przeniesienie informacji do obiektów:
 
 <Sandpack>
 
 ```js
 const drinks = {
-  tea: {
-    part: 'leaf',
-    caffeine: '15–70 mg/cup',
-    age: '4,000+ years'
+  herbata: {
```

Comment:
Mimo wszystko, ja bym nie tłumaczył nazw zmiennych/kluczy, mimo że JS to łyknie. Niech kod będzie angielski, a **wyświetlane** teksty polskie. Jeśli to konieczne, dodałbym kolejne warunki renderujące tekst, jak w komentarzu wyżej.